libs/s25main/ingameWindows/iwAIDebug.cpp

```diff
@@ -119,10 +118,11 @@ iwAIDebug::iwAIDebug(GameWorldView& gwv, const std::vector<const AIPlayer*>& ais
     overlays->AddString("Borderland");
     overlays->AddString("Fish");
 
-    text = AddText(ID_Text, DrawPoint(15, 120), "", COLOR_YELLOW, FontStyle::LEFT | FontStyle::TOP | FontStyle::NO_OUTLINE, NormalFont);
-
     // Show 7 lines of text and 1 empty line
-    SetIwSize(Extent(GetIwSize().x, text->GetPos().y + 8 * text->GetFont()->getHeight()));
+    text =
```

Comment:
this should be a separate issue

Reply:
Woops, yes. Slipped in as I worked on that before

Reply:
--> #1255